src/charm.py

```diff
@@ -157,6 +167,28 @@ def _csr_exists_in_requirer(self, csr: str, relation_id: int) -> bool:
                 return True
         return False
 
+    def _relation_id_parameter_valid(
+        self, found_relation_ids: List[int], requested_relation_id: str
```

Comment:
I'm not sure about the variable names. I would recommend:

- `found_relation_ids` -> `requirer_relation_ids`
- `requested_relation_id` -> `relation_id`

Reply:
The same applies elsewhere, especially related to the word `found`.

Reply:
Changed some names around.